app/models/user.rb

```diff
@@ -11,6 +10,14 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable, :confirmable
   attr_writer :login
 
+  def self.search(search)
```

Comment:
We will only come in this method if search term is present. So no need for conditional here.